smoke/features/jenkins-ephemeral.feature

```diff
@@ -5,6 +5,15 @@ Feature: Deploy Jenkins on openshift using template based install
 
     Background:
     Given Project [TEST_NAMESPACE] is used
+    Then we delete deploymentconfig.apps.openshift.io "jenkins"
+        And  we delete route.route.openshift.io "jenkins"
+        And  delete configmap "jenkins-trusted-ca-bundle"
+        And  delete serviceaccount "jenkins"
+        And  delete rolebinding.authorization.openshift.io "jenkins_edit"
+        And  delete service "jenkins-jnlp"
+        And  delete service "jenkins"
+        And delete all deploymentconfig
+        And delete all remaining test resources
```

Comment:
What about creating a single @step that does that in one shot?

Reply:
All the step definitions for the above are already present so didnt create a new one, as we use the same before starting the persistent template test